applications/conceptual-model-editor/src/diagram/diagram-controller.ts

```diff
@@ -412,14 +509,15 @@ const createActions = (
     //
     getSelectedNodes() {
       console.log("Diagram.getSelectedNodes");
-      return [];
+      return reactFlow.getNodes().filter(node => node.selected === true).map(node => node.data);
     },
     setSelectedNodes(nodes) {
       console.log("Diagram.setSelectedNodes", nodes);
     },
     getSelectedEdges() {
       console.log("Diagram.getSelectedEdges");
-      return [];
+      // Not sure why I have to check for undefined here, but in the case of selected nodes
```

Comment:
test if github works

Reply:
This was more of a reactflow's internal issue, since for some reason you can choose to not have data for edges but have to set data for nodes. 
![obrazek](https://github.com/user-attachments/assets/9e890c5b-0814-4b17-a797-65a425418fab)

![obrazek](https://github.com/user-attachments/assets/2be2201a-578f-4ec4-bac1-b9f19d28d4df)